tests/replication/_base.py

```diff
@@ -360,6 +361,10 @@ def make_worker_hs(
                 instance_loc.port,
                 lambda: self._handle_http_replication_attempt(worker_hs, port),
             )
+        elif instance_loc and isinstance(instance_loc, InstanceUnixLocationConfig):
+            raise Exception(
+                "Unix sockets are not supported for unit tests at this time."
```

Comment:
Possibly, but it would be fairly invasive I feel. Any place that references `tcpClient` and `tcpServer`(which there is only the one) would probably have to be abstracted a bit. I will consider it, but probably not for this PR as it could get kinda of 'sprawly'.

*Adds to TO-DO list for later*